geode-apis-compatible-with-redis/src/main/java/org/apache/geode/redis/internal/netty/NettyRedisServer.java

```diff
@@ -189,13 +189,15 @@ private void addSSLIfEnabled(SocketChannel ch, ChannelPipeline p) {
       return;
     }
 
+    if (sslConfigForServer.getKeystore() == null) {
+      throw new IllegalStateException(
+          "Cannot start netty as no key manager is configured. Please ensure that the Geode property 'ssl-keystore' is set.");
```

Comment:
I like throwing an exception here, but I'm not certain that "Geode property" is the best way to word it, in case this code gets used in some context that isn't specific to Geode.

Reply:
I'm not sure that's possible at this moment.

Reply:
even Geode docs refer to these as "gemfire properties" (see: https://geode.apache.org/docs/guide/113/reference/topics/gemfire_properties.html) so I would replace "Geode" with "gemfire" in this message.
Also if you do a google search for "gemfire property" it gets you to docs but "geode property" does not.

Reply:
Switched.